### PR TITLE
Add tips for the limitation of statements summary history table

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

The statement summary doc is not very clear: users are always confused why the data of the history table is becoming less.
I added tips for it to clarify why it may become less and how to solve this problem.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@djshow832 Please involve a technical review~

**Review by crazycs520 — COMMENTED** [review]
/lgtm

File: statement-summary-tables.md [hunk 1]
```diff
@@ -103,6 +103,10 @@ select * from employee where id in (...) and salary between ? and ?;
 - `tidb_stmt_summary_max_sql_length`：字段 `DIGEST_TEXT` 和 `QUERY_SAMPLE_TEXT` 的最大显示长度，默认值是 4096。
 - `tidb_stmt_summary_internal_query`：是否统计 TiDB 的内部 SQL。1 代表统计，0 代表不统计，默认不统计。
 
+> **注意：**
+>
+> 当一种 SQL 因为达到 `tidb_stmt_summary_max_stmt_count` 限制要被移除时，将会移除该 SQL 在所有时间段的数据。因此，即使一个时间段内的 SQL 种类数量没有达到上限，显示的 SQL 数量也会比实际的少。如遇到该情况，建议调大 `tidb_stmt_summary_max_stmt_count` 的值。
```
Comment (TomShawn) [inline statement-summary-tables.md:108]:
```suggestion
> 当一种 SQL 因为达到 `tidb_stmt_summary_max_stmt_count` 限制要被移除时，TiDB 会移除该 SQL 语句种类在所有时间段的数据。因此，即使一个时间段内的 SQL 种类数量没有达到上限，显示的 SQL 语句数量也会比实际的少。如遇到该情况，建议调大 `tidb_stmt_summary_max_stmt_count` 的值。
```

**Pull request merged**